app/controllers/buyers/users_controller.rb

```diff
@@ -1,17 +1,26 @@
+# frozen_string_literal: true
+
 class Buyers::UsersController < Buyers::BaseController
   activate_menu :audience, :accounts, :listing
 
-  inherit_resources
-  actions :index, :show, :edit
-  defaults :route_prefix => 'admin_buyers' #FIXME: inherited_resource makes us repeat this
-  belongs_to :account, :collection_name => :buyer_accounts
+  before_action :find_account, except: :index
+  before_action :find_user, except: :index
+
+  actions :show, :edit
```

Comment:
I think this is still inherited_resources, right?

Reply:
it could be 🤔 I will change it for `def show; end` and `def edit; end` then 😄